src/schema/metadata/AnatomicalLandmarkCoordinateSystem.yaml

```diff
@@ -0,0 +1,13 @@
+---
+key_name: AnatomicalLandmarkCoordinateSystem
+description: |
+  Defines the coordinate system for the anatomical landmarks.
+  Preferably the same as the `EEGCoordinateSystem`.
+  See Appendix VIII for a list of restricted keywords for coordinate systems.
+  If `Other`, provide definition of the coordinate system in
+  `AnatomicalLandmarkCoordinateSystemDescription`.
+anyOf:
+- $ref: common_definitions.json#/definitions/MEGCoordSys
+- $ref: common_definitions.json#/definitions/EEGCoordSys
+- $ref: common_definitions.json#/definitions/StandardTemplateCoordSys
+- $ref: common_definitions.json#/definitions/StandardTemplateDeprecatedCoordSys
```

Comment:
I'm not so sure about this one. Do we want to include references in the schema? If so, is this the best way to do it?

Reply:
Alternatively, we could make sure that all referenced definitions are in "private" files (e.g., `_ StandardTemplateCoordSys.yaml`), since those definitions shouldn't be useful on their own.